marked-element.html

```diff
@@ -200,36 +200,47 @@
         return;
       }
 
-      this.markdown = this._unindent(this._markdownElement.textContent);
+      if (this._markdownElement.textContent.trim() !== '') {
+        this.markdown = this._unindent(this._markdownElement.textContent);
+      }
 
       if (this._markdownElement.src) {
         this._request(this._markdownElement.src);
-      } else {
-        var observer = new MutationObserver(this._onScriptAttributeChanged
-            .bind(this));
-        observer.observe(this._markdownElement, { attributes: true })
       }
+
+      this._observer = new MutationObserver(this._onScriptAttributeChanged
```

Comment:
There is no `this._observer.observe`. This iwll only observe if attached is called after ready, and `ready` is not guaranteed to be called before `attached` (e.g. if this is held in memory for a long time before being attached or simply a race condition).